diff --git a/external/pico-sdk/lib/tinyusb/src/portable/raspberrypi/rp2040/rp2040_usb.c b/external/pico-sdk/lib/tinyusb/src/portable/raspberrypi/rp2040/rp2040_usb.c
index df05697fe..ed6194070 100644
--- a/external/pico-sdk/lib/tinyusb/src/portable/raspberrypi/rp2040/rp2040_usb.c
+++ b/external/pico-sdk/lib/tinyusb/src/portable/raspberrypi/rp2040/rp2040_usb.c
@@ -108,7 +108,7 @@ void __tusb_irq_path_func(_hw_endpoint_buffer_control_update32)(struct hw_endpoi
     {
       if ( *ep->buffer_control & USB_BUF_CTRL_AVAIL )
       {
-        panic("ep %d %s was already available", tu_edpt_number(ep->ep_addr), ep_dir_string[tu_edpt_dir(ep->ep_addr)]);
+       // panic("ep %d %s was already available", tu_edpt_number(ep->ep_addr), ep_dir_string[tu_edpt_dir(ep->ep_addr)]);
       }
       *ep->buffer_control = value & ~USB_BUF_CTRL_AVAIL;
       // 12 cycle delay.. (should be good for 48*12Mhz = 576Mhz)
