diff --git a/external/pico-sdk/src/rp2_common/hardware_xosc/include/hardware/xosc.h b/external/pico-sdk/src/rp2_common/hardware_xosc/include/hardware/xosc.h
index d49de4e..a044170 100644
--- a/external/pico-sdk/src/rp2_common/hardware_xosc/include/hardware/xosc.h
+++ b/external/pico-sdk/src/rp2_common/hardware_xosc/include/hardware/xosc.h
@@ -15,7 +15,7 @@
 
 // PICO_CONFIG: PICO_XOSC_STARTUP_DELAY_MULTIPLIER, Multiplier to lengthen xosc startup delay to accommodate slow-starting oscillators, type=int, min=1, default=1, group=hardware_xosc
 #ifndef PICO_XOSC_STARTUP_DELAY_MULTIPLIER
-#define PICO_XOSC_STARTUP_DELAY_MULTIPLIER 1
+#define PICO_XOSC_STARTUP_DELAY_MULTIPLIER 64
 #endif
 
 
